feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/data/repository/StakingRewardsRepository.kt

```diff
@@ -44,4 +47,12 @@ class StakingRewardsRepository(
         return stakingRewardDao.observeRewards(accountAddress)
             .mapList(::mapStakingRewardLocalToStakingReward)
     }
+
+    suspend fun stakingRewardSubQueryKusama(accountAddress: String): BigInteger {
+        return mapStakingSubquerySumRewardResponseToAmount(stakingApi.getSumRewardKusama(StakingSumRewardRequest(accountAddress = accountAddress)))
+    }
+
+    suspend fun stakingRewardSubQueryPolkadot(accountAddress: String): BigInteger {
+        return mapStakingSubquerySumRewardResponseToAmount(stakingApi.getSumRewardDot(StakingSumRewardRequest(accountAddress = accountAddress)))
```

Comment:
I would suggest to hide network requests separation in data layer. E.g. there should be only one method in StakingRewardsRepository, which will provide Flow<BigInteger> for any network